R/chk.R

```diff
@@ -101,41 +105,48 @@ chk_unique <- function(x) {
   chk_blank(x) | !(duplicated(x, fromLast = T) | duplicated(x, fromLast = F))
 }
 
+#' @rdname chk-generic
+#' @export
+chk_ascii <- function(x) {
```

Comment:
Possibly worth adding this one to the examples since it's not immediately obvious what it does. Looks like it's not exactly checking ASCII but rather printable ASCII

Reply:
Yep good call, thanks!

Reply:
Turns out `chk_ascii()` is broken! It's a dpautils relic...

Reply:
Oh good pickup. I actually didn't even realise this was in the package until I saw it highlighted in the changes